setup.py

```diff
@@ -27,7 +27,7 @@ def run(self):
             )
 
         if platform.system() == "Windows":
-            cmake_version = LooseVersion(
+            cmake_version = parse(
                 re.search(r"version\s*([\d.]+)", out.decode()).group(1)
             )
             if cmake_version < "3.1.0":
```

Comment:
I think you need to parse the RHS as well here.

Reply:
what do you mean?